deliverables/README.md

```diff
@@ -2,7 +2,10 @@
 ###Intro to the Back End
 ####Required
 What is the difference between front- and back-end development?<br>
+The front-end of a website is the part that users interact with. In order to execute user input and build an immersive experience, front-end developers write in HTML, CSS, and Javascript. In addition to fluency in these languages, front-end devs need to be familiar with frameworks that will ensure great-looking content no matter the device, and libraries to package code into a more useful, time-saving form. The back-end of a website consists of a server, an application, and a database. Back-end developemers use server side languages like Ruby to build an application and find/save/change the data that a user inputs then output a response back to the user at the front-end.
+
```

Comment:
Good :+1:  Just to be technical think of find/save/change as CRUD or Create, Read, Update and Destroy